Minor updates in the calibration procedure #2

lapandic opened the pull request:
added local:=true flag, fixed some commands to follow procedures established in previous chapters

liampaull commented:
let's not use cd ~/duckietown any more instead cd ![DUCKIETOWN_ROOT]

the ground_projection node should be run on the laptop not the robot

lapandic commented:
Duckietown root on duckiebot have to be ~/duckietown and it is because we force them to make it like that.
Ground_projection is indeed running on the laptop by this instructions. 
Also, I have changed name of step 7 to step 4, because steps 4-6 were deleted and fixed step 1 in extrinsic calibration since it doesn't look so pretty right now.
